apis/cloud.redhat.com/v1alpha1/clowdapp_types.go

```diff
@@ -496,7 +494,15 @@ func (i *ClowdApp) GetCronJobNamespacedName(d *Job) types.NamespacedName {
 
 // IsReady returns true when all the ManagedDeployments are Ready
```

Comment:
Is this comment true anymore?  Even if it is, maybe it would be more accurate to say we are just delegating to the ready bool, assuming reconciliation has been successful